### PR TITLE
FIX: Handle CXXFLAGS environment variable on stable/2.9.0

### PR DESCRIPTION
**Pull request by reneme**
This is the same patch like [this one](https://github.com/bincrafters/conan-botan/pull/6), just applied to `stable/2.9.0`.

### CODE REVIEW

**Review by uilianries — APPROVED** [review]
LGTM. Let's wait for CI result before to merge. Thanks again!

**Comment by reneme** [comment]
... not sure what happened. The log seems to be truncated... does appveyor kill long-running jobs at some point?

**Comment by uilianries** [comment]
I dunno, I've just restarted, because the log is truncated indeed, even downloading the full log.

**Pull request merged**
